### ISSUE TITLE
Some png image icons look poor

### ISSUE THREAD

**Issue by Aybee** [gmaps-marker-clusterer/gmaps-marker-clusterer#55]
Original ticket was opened here https://github.com/googlemaps/js-marker-clusterer/issues/139

Seems that some of the icons need to get newly rendered with nice alpha transparency. See here
https://github.com/delahaye/dlh_googlemaps/issues/68

**Comment by XmlmXmlmX** [comment]
Hi, you mean [these one here](https://github.com/gmaps-marker-clusterer/gmaps-marker-clusterer/tree/master/images), right?
To be honest I have never used these. For me these are just sample files.
The problem is that we have no source files. They were not included in Google's original repsitory.

**Comment by XmlmXmlmX** [comment]
I would recommend to use custom svg icons to avoid rendering issues: https://stackoverflow.com/questions/24413766/how-to-use-svg-markers-in-google-maps-api-v3

**Comment by Aybee** [comment]
Yes I mean those files. SVG would be nice and in PNG als default ones. If e.g. a user adds a plugin to a CMS, there should be nice icons by default.
